tests/scripts/task_ci_python_setup.sh

```diff
@@ -31,3 +31,4 @@ set -o pipefail
 echo "Addtiional setup in" ${CI_IMAGE_NAME}
 
 python3 -m pip install --user tlcpack-sphinx-addon==0.1.3 synr==0.2.1
+python3 -m pip install --user tokenizers==0.9.4 transformers==4.0.1
```

Comment:
Is this necessary?

Reply:
@tkonolige : I am getting a strange error in CI when it is trying to build for docs. I thought the necessary packages were missing, so trying with the CI script, but to in-vain. Would you please check once, i am not sure what i am missing here. TIA!

Reply:
I'm not sure which commit was broken. Can you remove these and then we can debug?

Reply:
Okay will revert this change and check the error.

Reply:
@tkonolige : I think the error is reproduced now. Would you please have a look once in the latest CI report.

Reply:
Ok, I think I have this figured out. This tutorial depends on transformers, but because the import was local to the download function (which wasn't run), we didn't hit it. That means that we need to install transformers in the ci_script (like you had done before). However, this model is pretty large (500MB), so I don't think we want to be downloading it for every run on CI. Do you have a small sparse model we could use instead? If not, I suggest we leave the finally running of the script commented out and let the user run it if they so choose.

Reply:
Thanks @tkonolige ! I will look into it further.